tasks/download_kubeconfig.yml

```diff
@@ -0,0 +1,21 @@
+---
+
```

Comment:
Missing tasks:

 - Task to ensure destination directory exists
 - Task(s) to check if the file needs to be downloaded again to avoid breakage of idempotence

Reply:
Missing tasks added